asm/istio/expansion/vm-eastwest-gateway.yaml

```diff
@@ -0,0 +1,46 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+metadata:
+  name: eastwest
+spec:
+  profile: empty
```

Comment:
Don't specify the profile, this will be overlaid on the base operator which already specifies it.

Reply:
This is needed. As we do not want to override the existing istio installation, so empty profile here will help only install eastwest gateway.

Reply:
Did you test that if you overlay this on top of an operator that specifies a profile that's not empty that it doesn't overwrite the profile choice?

Or is this file always applied in a separate `istioctl install` command than the main one? If so, please document that in the README.

Reply:
This file is applied separately using istioctl after the control plane is installed.